lib/v2/shuiguopai/index.js

```diff
@@ -6,7 +6,7 @@ const { art } = require('@/utils/render');
 const path = require('path');
 
 module.exports = async (ctx) => {
-    const rootUrl = 'https://shuiguopai.com';
+    const rootUrl = 'https://sgp3.fun';
```

Comment:
```suggestion
    const rootUrl = 'http://shuiguopai.com';
```
It works after switching to HTTP.